docs/src/howto/install/charm.md

```diff
@@ -0,0 +1,87 @@
+# Install Canonical Kubernetes from a charm
+
+Canonical Kubernetes is packaged as a [charm], available from the 
+snap store for all supported platforms.
+
+## What you'll need
+
+This guide assumes the following:
+
+- The rest of this page assumes you already have Juju installed and have added [credentials] for a cloud and bootstrapped a controller.
+- If you still need to do this, please take a look at the quickstart instructions, or, for custom clouds (OpenStack, MAAS), please consult the Juju documentation.
```

Comment:
This is all part of the previous requirement - Also we should link to the Juju install page (https://juju.is/docs/juju/install-juju)